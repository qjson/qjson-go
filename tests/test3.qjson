v1 : (1h 2m 2s) * 2
v2 : 1.3 + 1h
v3 : 2m 15
v4 : 2m + 15
v5 : 1m 15 + 3 * 2
v6 : 2 * 1h
v7 :Â 1.3 + 60
v8 : (2+3)*2 
v9 : 1.4 + 5.2
v10: 5 + 1.3 