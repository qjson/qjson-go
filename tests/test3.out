{"v1":7444,"v2":3601.3,"v3":135,"v4":135,"v5":81,"v6":7200,"v7":61.3,"v8":10,"v9":6.6,"v10":6.3}
